Sources/Valet/SecureEnclaveAccessControl.swift

```diff
@@ -75,17 +75,17 @@ public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable
         case .userPresence:
             return .userPresence
         case .biometricAny:
-            if #available(macOS 10.12.1, *) {
-                return .init(rawValue: 2) // .biometryAny with Xcode 9 compatibility.
+            if #available(tvOS 11.3, OSXApplicationExtension 10.13.4, macOS 10.12.1, *) {
```

Comment:
When I started using `biometryAny` in the code, I got a build error that I needed a `#available` check for `OSXApplicationExtension 10.13.4`. I added it, and updated the documentation above.

Same comment applies to `biometryCurrentSet` below.